ads/google/a4a/traffic-experiments.js

```diff
@@ -76,6 +82,14 @@ export function googleAdsIsA4AEnabled(win, element, experimentName,
       // Page is selected into the overall traffic experiment.
       const selectedBranch = getPageExperimentBranch(win, experimentName);
       addExperimentIdToElement(selectedBranch, element);
+      // Detect how page was selected into the overall experimentName.
+      if (selectedBranch == externalBranches.experiment ||
```

Comment:
At some point we'll want to remove the other experiment branches in which case this will no longer work.  Instead why not have maybeSetExperimentFromUrl return true or false depending on if selection was successful and then use that to select the new experiment IDs?